src/interp/binary-reader-interp.cc

```diff
@@ -620,7 +620,47 @@ Result BinaryReaderInterp::BeginGlobal(Index index, Type type, bool mutable_) {
   return Result::Ok;
 }
 
+Result BinaryReaderInterp::BeginGlobalInitExpr(Index index) {
+  return BeginInitExpr();
+}
+
+Result BinaryReaderInterp::EndInitExpr() {
+  assert(reading_init_expr_);
```

Comment:
can this somehow arise from a malformed binary? or is this an invariant that's maintained by wabt?
edit: I guess the superclass guarantees it?

Reply:
Yup, it would be  bug in wabt for this to fire I think.